src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__title {
+  width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px 4px;
+  color: #060b35;
+}
+
+.card__code {
+  width: fit-content;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 84px 16px 16px;
+}
+
+.card__rating {
+  width: 166px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.stars-star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star.svg);
+
+  display: inline-block;
+}
+
+.stars .stars-star:nth-child(-n + 4) {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star-active.svg);
+
+  display: inline-block;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  align-self: center;
+  color: #060b35;
+}
+
+.card__price-block {
+  width: 166px;
+  display: flex;
+  justify-content: space-between;
+  height: 18px;
+  align-items: center;
+  margin: 0 16px 16px;
+}
+
+.card__price-title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  align-self: center;
+  color: #060b35;
+}
+
+.card__buy-button {
+  width: 166px;
+  height: 40px;
+  margin: 0 16px 16px;
+
+  border-radius: 5px;
+  border: #00acdc;
```

Comment:
The 'border' property should include the border-style (e.g., solid), not just the color. It seems like there's a mistake here because only the color is specified. According to the checklist, the border property should be written shorthand with all its parts.